### ISSUE TITLE
Secure Software Supply Chain Aspects

### ISSUE THREAD

**Issue by afrittoli** [cdevents/spec#70]
Investigate S3C aspects for CDEvents.

Examples: artifact SBOM, signature, artifact verified event

**Comment by e-backmark-ericsson** [comment]
From the SIG Events meeting on Oct 10th:
- One idea is also to include a reference to an in-toto attestation for a built artifact.
- Signing all events to make sure they are not tampered with since being created

**Comment by afrittoli** [comment]
We may be able to address parts of this in v0.2, for instance, add a "signed" predicate for the "artifact" event.

**Comment by afrittoli** [comment]
Signed predicated defined in https://github.com/cdevents/spec/pull/125
Once this PR is merged, we can open a new issue to track further work in this area
